chore: readme major update (#696)
revanced/revanced-manager: README.md

<p align="center">

>   <img width="800" src="Logo.png">

Reviewer: This doesn't follow any style guide. Please refer to [this](https://cdn.discordapp.com/attachments/952987428786941952/1067552057940267079/revanced-brand-identity.zip).

Reviewer 2: What is this 😵‍💫😵‍💫

Btw OsuMatrix, Circular logo where?

Reviewer: Masked off the full logo in the zip.

Reviewer 2: This is not even the logo i updated 🙁
It still the old one

Reviewer: I can't find the updated one, also it's not open source, might want to create a repository for it.

Reviewer 2: [Here](https://cdn.discordapp.com/attachments/953965039105232906/1070354383159709798/revanced-brand-identity.zip)

Author: Sorry for the inconvenience. I chose the logo from the `res` folder in the main code. I will make sure to abide by the official logos

Reviewer 2: > Sorry for the inconvenience. I chose the logo from the `res` folder in the main code. I will make sure to abide by the official logos.

Logo & typo both

Reviewer 2: @oSumAtrIX what about this wordmark?
![revanced-manager-wordmark](https://user-images.githubusercontent.com/82371061/218303047-45f601be-9e67-4cf5-ad2d-0fed47c4bbbd.svg)

> </p>

Reviewer 3: The logo can be improved by changing the font to [Manrope](https://github.com/sharanda/manrope "Manrope font on GitHub"), Open Sans, or similar. See my example design for reference. *you can use this design, there's nothing stopping you :) but [see this.](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to)*

![ReVanced Manager Valentine - Light mode](https://user-images.githubusercontent.com/93124920/218297719-bceb269d-8abc-4007-be5e-59c756513b48.png "ReVanced Manager banner - Valentine themed - Light mode")

![ReVanced Manager Valentine - Dark mode](https://user-images.githubusercontent.com/93124920/218297710-30f88511-fa37-4c93-b37b-42438ef58829.png "ReVanced Manager banner - Valentine themed - Dark mode")

Reviewer 2: Why font needs to change? 
Why not follow the brand guide?

Reviewer 2: ![revanced-manager-wordmark](https://user-images.githubusercontent.com/82371061/218303155-28b9df3e-9c74-4f34-b70c-247af9586695.svg)

Reviewer: We currently lack an extensive kit of our current language design. A repository should be created to discuss this matter openly.

Reviewer 2: > Why font needs to change? Why not follow the brand guide?

I just realize Manrope is the font used on ReVanced wordmark. My bad

### <p align="center">  The official ReVanced Manager based on Flutter</p>

> **Warning**: This repository currently has no active maintainer. For this reason, development is stale. Active development continues for [revanced-cli](https://github.com/revanced/revanced-cli). If you are interested in maintaining this repository, please let us know at manager@revanced.app.

## 🔽 Download

> ReVanced Manager is still in early development (ALPHA) . You can install the latest version [here](https://github.com/revanced/revanced-manager/releases/latest).

Reviewer: ```suggestion
ReVanced Manager is still in early development. You can obtain the latest version [here](https://github.com/revanced/revanced-manager/releases/latest).
```

## 📝 Prerequisites
1. Android 8 or higher.

> 2. [Vanced MicroG](https://github.com/TeamVanced/VancedMicroG/releases) is required for YouTube and YouTube Music (only for non-root).

Reviewer: This is not a prerequisite.

Author: I didn't change any prerequisite. These are the ones written in the `flutter` branch.

Reviewer: It's not correct in the `flutter` branch.

> &nbsp; 🟥 Does not work on some ARMv7 devices

Reviewer: This is inconsistent markdown. Use lists.

Author: Not working on ARMv7 devices is not a prerequisite so I thought that including it in the list doesn't make any sense

Reviewer: In that case it doesn't belong in the section at all and should be moved outside

For suggestions and bug reports, open an issue [here](https://github.com/revanced/revanced-manager/issues/new/choose).

## 💭 Discussion

> If you wish to discuss the Manager, a thread has been made under the [#development](https://discord.com/channels/952946952348270622/1002922226443632761) channel in the Discord server. *This thread may be removed in the future.*

Reviewer: ```suggestion
If you wish to discuss about ReVanced Manager, a thread has been made under the [#development](https://discord.com/channels/952946952348270622/1002922226443632761) channel on the ReVanced Discord server. *This thread may be closed in the future.*
```

## 🌐 Translation
[![Crowdin](https://badges.crowdin.net/revanced/localized.svg)](https://crowdin.com/project/revanced)

> We are accepting translations on [Crowdin](https://translate.revanced.app)

Reviewer: ```suggestion
Translations can be made on [Crowdin](https://translate.revanced.app)
```

> ## 🛠️ Building the Manager from the source code

Reviewer: ```suggestion
## 🛠️ Building ReVanced Manager from source
```

> 1. Setup flutter environment for your [platform.](https://docs.flutter.dev/get-started/install)

Reviewer: Usually, you don't end lists with a punctuation mark.

2. Clone the repository locally.
3. Add your github token in gradle.properties [like this](https://github.com/revanced/revanced-manager/blob/docs/docs/5_building-from-source.md).

> 4. Open the project in the terminal.

Reviewer: ```suggestion
4. Open the project in a terminal.
```
